app/src/main/java/org/jellyfin/androidtv/itemhandling/ItemRowAdapter.java

```diff
@@ -812,7 +813,7 @@ public void onResponse(ItemsResult response) {
                     for (BaseItemDto item : response.getItems()) {
                         //re-map the display prefs id to our actual id
                         item.setDisplayPreferencesId(item.getId());
-                        if (!ignoreTypeList.contains(item.getCollectionType()) && !ignoreTypeList.contains(item.getType())) {
```

Comment:
Why was the second half of this check removed?

Reply:
IIRC it could never be true, so I removed it. I'll check again.

Reply:
So, `ignoreTypeList` is initialized as following:

```
    private static String[] ignoreTypes = new String[]{"books", "games"};
    private static List<String> ignoreTypeList = Arrays.asList(ignoreTypes);
```

and I think those are collection types (which I should probably turn into enums as well in a future PR), and doesn't contain any item types. Another case where the usage of strings as types confuses things.

Reply:
Good catch! Support for games has also been removed from the server so we should be able to remove it at some point also.